cfme/intelligence/chargeback.py

```diff
@@ -144,119 +178,69 @@ def _fill_assignform_dict(form, d):
-             'mem_used': self.mem_used,
-             'mem_used_var': self.mem_used_var,
-             'net_io': self.net_io,
-             'net_io_var': self.net_io_var},
-            action=rate_form.add_button)
-        flash.assert_no_errors()
+        view = navigate_to(self, 'New')
+        fields = self.fields
+        view.fill_with({'description': self.description,
+                        'currency': self.currency,
+                        'fields': fields},
+                       on_change=view.add_button,
+                       no_change=view.cancel_button)
+
+        view.flash.assert_success_message('Chargeback Rate "{}" was added'.format(
+            self.description))
 
     def update(self, updates):
-        navigate_to(self, 'Edit')
-        fill(rate_form,
-            {'description': updates.get('description'),
-             'cpu_alloc': updates.get('cpu_alloc'),
-             'cpu_used': updates.get('cpu_used'),
-             'disk_io': updates.get('disk_io'),
-             'compute_fixed_1': updates.get('compute_fixed_1'),
-             'compute_fixed_2': updates.get('compute_fixed_2'),
-             'mem_alloc': updates.get('memory_allocated'),
-             'mem_used': updates.get('memory_used'),
-             'net_io': updates.get('network_io')},
-            action=rate_form.save_button)
-        flash.assert_no_errors()
+        view = navigate_to(self, 'Edit')
+        view.fill_with(updates, on_change=view.save_button, no_change=view.cancel_button)
+        view.flash.assert_success_message('Chargeback Rate "{}" was saved'.format(
+            updates.get('description')))
 
     def delete(self):
-        navigate_to(self, 'Details')
-        tb_select('Remove from the VMDB', invokes_alert=True)
-        sel.handle_alert()
-        flash.assert_no_errors()
+        view = navigate_to(self, 'Details')
+        view.configuration.item_select('Remove from the VMDB', handle_alert=True)
+        view.flash.assert_success_message('Chargeback Rate "{}": Delete successful'.format(
+            self.description))
 
 
 @navigator.register(ComputeRate, 'All')
 class ComputeRateAll(CFMENavigateStep):
+    VIEW = ChargebackView
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
         self.prerequisite_view.navigation.select('Cloud Intel', 'Chargeback')
         accordion.tree("Rates", "Rates", "Compute")
 
     def am_i_here(self):
-        return match_page(summary="Compute Chargeback Rates")
+        return self.view.in_explorer
 
 
 @navigator.register(ComputeRate, 'New')
 class ComputeRateNew(CFMENavigateStep):
+    VIEW = AddComputeChargebackView
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        tb_select("Add a new Chargeback Rate")
+        self.view.configuration.item_select("Add a new Chargeback Rate")
 
 
 @navigator.register(ComputeRate, 'Details')
 class ComputeRateDetails(CFMENavigateStep):
+    VIEW = ChargebackView
```

Comment:
same view like in All destination and am_I_here present, can you config this is correct and expected ?